app/presenters/hyku/work_show_presenter.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Hyku
```

Comment:
This looks like something to contribute back to Hyku Prime.  And might be there already.